src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -377,14 +399,19 @@ public EmitOptions WithIncludePrivateMembers(bool value)
             return new EmitOptions(this) { IncludePrivateMembers = value };
         }
 
-        public EmitOptions WithInstrument(string instrument)
+        public EmitOptions WithInstrument(params InstrumentationKind[] instrument)
+        {
+            return WithInstrument(ImmutableArray.Create(instrument));
+        }
+
+        public EmitOptions WithInstrument(ImmutableArray<InstrumentationKind> instrument)
```

Comment:
+1

---
In reply to: [75935486](https://github.com/dotnet/roslyn/pull/13013#discussion_r75935486) [](ancestors = 75935486)